### PR TITLE
Fix push notifications + LMR-1312, LMR-1313, LMR-1314

### PR DESCRIPTION
**Pull request by kubedo8**

### CODE REVIEW

File: src/app/core/store/documents/documents.effects.ts [hunk 1]
```diff
@@ -354,37 +318,3 @@ export class DocumentsEffects {
     private store$: Store<AppState>
   ) {}
 }
-
-function createSyncCollectionAction(
```
Comment (livthomas) [inline src/app/core/store/documents/documents.effects.ts]: I like the fact that we do not need this hacking anymore.

File: src/environments/environment-variables.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ const locale: string = env['I18N_LOCALE'] || 'en';
 const sentryDsn: string = env['SENTRY_DSN'];
 const sessionTimeout = Number(env['SESSION_TIMEOUT']) || 30; // minutes
 const pusherCluster = env['PUSHER_CLUSTER'] || 'eu';
-const pusherKey = env['PUSHER_KEY'] || '';
+const pusherKey = env['PUSHER_KEY'] || '22b1d531f1fc10719c7c';
```
Comment (livthomas) [inline src/environments/environment-variables.ts]: Do we want to have this in the repository?

Comment (kubedo8) [reply src/environments/environment-variables.ts]: I will definitely remove it!

File: src/app/core/store/collections/collections.reducer.ts [hunk 3]
```diff
@@ -67,14 +66,40 @@ export function collectionsReducer(
...
 }
 
+function addCollections(state: CollectionsState, collections: Collection[]): CollectionsState {
+  const newState = {...state, loaded: true};
+  const filteredCollections = collections.filter(collection => {
+    const oldCollection = state.entities[collection.id];
+    return !oldCollection || isCollectionNewer(collection, oldCollection);
+  });
+
+  return collectionsAdapter.addMany(filteredCollections, newState);
```
[the remark this answers is not in the document]
Comment (kubedo8) [reply src/app/core/store/collections/collections.reducer.ts]: You are right, but I want to prepare it for the future. Today we talked about force refresh of resources, so maybe we can use this approach. What do you think?

File: src/app/core/dto/organization.dto.ts [hunk 4]
```diff
@@ -17,9 +17,9 @@
...
 
-import {Project} from './project';
-import {Resource} from './resource';
+import {ProjectDto} from './project.dto';
+import {ResourceDto} from './resource.dto';
 
-export interface Organization extends Resource {
-  projects?: Project[];
+export interface OrganizationDto extends ResourceDto {
+  projects?: ProjectDto[];
```
[the remark this answers is not in the document]
Comment (kubedo8) [reply src/app/core/dto/organization.dto.ts]: Ok

**Pull request merged**